.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: ci
+
+on: pull_request
+
+jobs:
+
+  frontend:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+        env:
+          RUNNER_TEMP: /tmp
```

Comment:
Do we need this in GitHub Actions or was it only for act?

Reply:
Can't remember for sure. Will try taking it out.